README.md

```diff
@@ -938,6 +938,15 @@ We recommend you specify exact versions of lint libraries, including `tslint-mic
       </td>
       <td>2.0.11</td>
     </tr>
+    <tr>
+      <td>
+        <code>react-a11y-mouse-event-has-key-event</code>
+      </td>
+      <td>
+        For accessibility of your website, elements with mouseOver/mouseOut should be accompanied by onFocus/onBlur keyboard events.
+      </td>
+      <td>6.1.0</td>
```

Comment:
@JoshuaKGoldberg what is the next version. I think I need to catch this branch up to master.

Reply:
Also how do you generate the `tslint-warnings.csv` from this? I know I did this before 🤔

Reply:
@lizzzp1 new rules should have `@next` in version column. It will be replaced during development.

csv file should be created from rule class metadata.

Reply:
@IllusionMH ah, got it. Thanks!